packages/koa-joi-validate/package.json

```diff
@@ -40,15 +40,21 @@
     "@types/koa": "^2.0.48",
     "boom": "^7.3.0",
     "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
     "chai-http": "^4.2.1",
-    "joi": "^10.6.0",
+    "joi": "10.6.0",
+    "joi10": "npm:joi@^10.6.0",
```

Comment:
Maybe we don't need to add both joi & joi10 as joi's version is frozen anyway

Reply:
joi10 and joiX are used for testing purpose and won't move, while joi dep can be updated and is the version used in the package, not in test suite.